controllers/ovndbcluster_controller.go

```diff
@@ -579,8 +570,110 @@ func (r *OVNDBClusterReconciler) reconcileServices(
 			if err != nil {
 				return ctrl.Result{}, err
 			}
+			// Delete DNS records for deleted services/pods
+			dnsRecord := &infranetworkv1.DNSData{}
```

Comment:
let's be consistent with usage with variables, as dnsRecord used here for DNSData and later for DNSHost, can use dnsData for DNSData and dnsHost for DNSHost